### ISSUE TITLE
Add language files support

### ISSUE THREAD

**Issue by inxomnyaa** [PEMapModder/HereAuth#2]

**Comment by PEMapModder** [comment]
What do you mean by that, exactly? Per player language or per server language?

Because it is hard to try to know the language of a player when he is registering.

**Comment by inxomnyaa** [comment]
Well.. the second thing. Per server. Because if you have spanish kids that can't read english..

**Comment by PEMapModder** [comment]
Then how do they know how to read English to choose a Spanish version of the server?

**Comment by PEMapModder** [comment]
Also, for the sake of management convenience, I prefer to create one config file per version new configurations.

**Comment by dancing-ipsum** [comment]
for example use /lang ____ to change language used in default configurations

**Comment by PEMapModder** [comment]
And how do you expect them to know such command?
And what happens if every plugin uses /lang? Only one plugin will work.

**Comment by dancing-ipsum** [comment]
Not many plugins use /lang actually.

**Comment by PEMapModder** [comment]
As long as there is more than one plugin, it is incorrect.
I guess we can detect language from server configuration.

**Comment by jarne** [comment]
You could get the language by IP like Lifeboat does.

**Comment by PEMapModder** [comment]
@SURVAml that can be highly inaccurate.

**Comment by inxomnyaa** [comment]
Just 1 language for a server. If you want to go save, use english. Done.

**Comment by Thunder33345** [comment]
agreed @PEMapModder i have ask.fm for displaying stuff in malay; user's country languge != user language preference
and that just backfired a lot when traveling

i suggest using something like
/$mastercmd lang
/$mastercmd setlang
so assume $mastercmd is HereAuth
i dont think any plugin uses the cmd /hereauth so that will resolve all command issues 

new issue:
what language should server display on player first join

**Comment by PEMapModder** [comment]
@Thunder33345 that's why there should only be one language for the whole server.

**Comment by PEMapModder** [comment]
What @thebigsmileXD suggested was to separate messages into another language file.

**Comment by dancing-ipsum** [comment]
or change default language in config.yml

**Comment by dancing-ipsum** [comment]
that will resolve the command problem

**Comment by PEMapModder** [comment]
@MCPEGamerJPatGitHub the purpose of separating messages is to allow easy switching of server's plugin language.

**Comment by dancing-ipsum** [comment]
@pemapmodder But commands will be a mess, when HereAuth has many configurations in the config file, making it very confusing for some users

**Comment by PEMapModder** [comment]
how does commands have anything to do with config?

**Comment by dancing-ipsum** [comment]
users might be confused
